airflow/settings.py

```diff
@@ -188,17 +188,26 @@ def configure_adapters():
         pass
 
 
+def configure_action_logging():
+    """
+    Any additional configuration (register callback) for airflow.utils.action_loggers
+    module
+    :return: None
+    """
+    pass
+
+
 try:
     from airflow_local_settings import *
-
     log.info("Loaded airflow_local_settings.")
-except:
-    pass
+except Exception as e:
+    logging.debug('Failed to load airflow_local_settings', exc_info=1)
```

Comment:
Having an `airflow_local_settings` is not a requirement, this may be confusing to new users. If the default logging level is debug I'd keep the previous behavior.

Reply:
Good point. The default level is INFO from settings, but I think it could confuse the user with the stack trace. Let me keep the previous behavior.